.travis.yml

```diff
@@ -20,10 +21,17 @@ script:
   - set -o pipefail
   - xcodebuild -version
   - xcodebuild -showsdks
+  - if [[ ${GREY_RUBY} ]]; then
+      cd gem;
+      bundle install --retry=3;
```

Comment:
Rails, a well known Ruby project, [uses the same setting.](https://github.com/rails/rails/blob/d6e7334d19c234d42904e4a70b5a76831731b114/.travis.yml#L18)